source/builders.txt

```diff
@@ -0,0 +1,117 @@
+.. _kotlin-sync-builders:
+
+===============================
+Implement Builders Code Pattern
+===============================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. .. toctree::
+
+..    /fundamentals/builders/aggregates
+..    /fundamentals/builders/filters
+..    /fundamentals/builders/indexes
+..    /fundamentals/builders/projections
+..    /fundamentals/builders/sort
+..    /fundamentals/builders/updates
+
+Overview
+--------
+
+This page describes how to implement the builders pattern in your code and describes
+benefits of using builder classes.
+
+The {+driver-short+} provides type-safe builder classes and methods that enable developers to
+efficiently build queries and aggregations. To learn more about the builders pattern,
+see `Type-safe builders <https://kotlinlang.org/docs/type-safe-builders.html>`__ in the
+{+language+} documentation.
```

Comment:
```suggestion
This page describes how to use the various available builders in your code and describes
benefits of using the provided builders.

The {+driver-short+} provides type-safe builder classes and methods that enable developers to
efficiently build queries and aggregations.
```

Reply:
Note: https://kotlinlang.org/docs/type-safe-builders.html may not be appropiate here, as the builders used are written in Java and don't/can't catch the type information in the same way as the examples in the kotlin documentation.